CHALLENGE 2.0/numbers_that_form_triangle.py

```diff
@@ -0,0 +1,11 @@
+def triangle_num(a, b, c):
```

Comment:
You always assume your function would receive a happy-path parameter. Could you validate the inputs here before doing the calculation?

Reply:
i dont grab this. By validate do you mean i  should make the function accept other data type?